skimage/feature/texture.py

```diff
@@ -291,3 +293,130 @@ def local_binary_pattern(image, P, R, method='default'):
     image = np.ascontiguousarray(image, dtype=np.double)
     output = _local_binary_pattern(image, P, R, methods[method.lower()])
     return output
+
+def multiblock_local_binary_pattern(int_image, x, y, width, height):
+    """Multi-block local binary pattern.
+
+    MB-LBP is an extension of LBP that can be computed on many
+    scales in a constant time using integral image. It consists of
+    9 equal-sized rectangles. Sum of pixels' intensity values
```

Comment:
"9 equally sized rectangles" (or should it be squares?)  The mean pixel value in each rectangle is compared to that of the central rectangle, and the feature descriptor is calculated according to their comparative values.

Alternatively, you can say something like:

The features are calculated in a way similar to local binary patterns, except that block means rather than pixel values are used.